src/PKSim.Core/Services/ParameterTask.cs

```diff
@@ -371,16 +371,7 @@ private ICommand withUpdatedDefaultStateAndValue(IOSPSuiteCommand executedComman
          macroCommand.Add(executedCommand);
          macroCommand.Add(new SetParameterDefaultStateCommand(parameter, isDefault: false) {ShouldChangeVersion = shouldChangeVersion}.Run(_executionContext).AsHidden());
 
-         if (!parameter.ValueOrigin.IsUndefined)
```

Comment:
This is the big code change here. Instead of only updating for undefined, we update all the time using the ValueOrigin.Unknown